Dockerfile

```diff
@@ -2,6 +2,18 @@ FROM hashicorp/terraform:0.11.10
 
 ENV DOCKER_VERSION=18.06.1-ce
 
+# Install aws cli
+RUN apk -v --update add \
+    python \
+    py-pip \
+    groff \
+    less \
+    mailcap \
+    && \
+    pip install --upgrade awscli==1.14.5 s3cmd==2.0.1 python-magic && \
```

Comment:
1. awscli 1.4.5 is an old version
2. aws s3 is a valid command I'm using with the CLI already